README.md

```diff
@@ -194,6 +194,27 @@ PasswordEncrypted: '@@FILE:/dir/@@PROP:name@@.txt@@'
 PasswordEncrypted: '@@FILE:@@ORACLE_HOME@@/dir/name.txt@@'
 ```
 
+**Environment tokens** are declared with the syntax `@@ENV:<name>@@`. This type of token is resolved by looking up the system environment variable `<name>`, and substituting that value for the token.
+
+**Secret tokens** are declared with the syntax `@@SECRET:<name>:<key>@@`. This type of token is resolved by determining the location of a Kubernetes secret file, and reading the first line from that file. That line is substituted for the token.
+
+There are two methods for deriving the location of the Kubernetes secret file. The first method involves using one or more configured root directories, and looking for the secret file in the paths `<root-directory>/<name>/<key>`.
+
+The root directories are configured as a comma-separated list of directories, using the environment variable `WDT_MODEL_SECRETS_DIRS`. For example, `WDT_MODEL_SECRETS_DIRS` is set to `/etc/my-secrets,/etc/your-secrets`, the token `@@SECRET:secrets:the-secret@@` will search the following locations:
```

Comment:
For example, `WDT_MODEL_SECRETS_DIRS` is set to `/etc/my-secrets,/etc/your-secrets`, the token -> For example, if `WDT_MODEL_SECRETS_DIRS` is set to `/etc/my-secrets,/etc/your-secrets`, then the token